docs/using-docker.md

```diff
@@ -0,0 +1,38 @@
+# Using Docker for job execution
+
+## Overview 
+Turbinia can support Docker by allowing users to configure jobs to execute through a Docker container. This eliminates the need to install all the necessary dependencies and/or external programs on the host machine and instead execute the job through a container that has all the necessary dependencies installed. Please note that Turbinia does not provide the Docker images necessary for each job and they will need to either be created or pulled from a container registry. 
+
+## Enabling Docker usage
+In order to enable this feature, please take the following steps. 
+1. Install the Docker daemon on the host machine. Please visit the Docker website for the [Installation Guide](https://docs.docker.com/install/).
+2. In the `.turbiniarc` configuration file, set the `DOCKER_ENABLED` flag to `True` to enable the usage of Docker. 
+3. Review the `DEPENDENCIES` flag in the `.turbiniarc` configuration file and identify which job you would like to execute a Docker container for. Once identified, replace the value for `docker_image` with the `image_id` of the Docker image. 
+4. Save the `.turbiniarc` configuration file then start a new worker.
+5. If the dependency check succeeds, the worker should now execute the configured job through a Docker container. 
```

Comment:
The reader might not be aware of what the dependency check is here, so you might say something like:
`When the Workers start, they will perform dependency checks to ensure that the binaries required by the Job are installed in the container, and if that check passes it will execute those in the configured Docker container.`